src/containers/App.tsx

```diff
@@ -117,9 +109,6 @@ export const App: FunctionComponent = () => {
                         screenSize={screenSize}
                       />
                     </div>
-                    {displayByFranchise && (
```

Comment:
isn't this needed anymore?

Reply:
No, Franchise was moved into selected filters ccomponent.